test/content/audio/following_train_test.exs

```diff
@@ -12,22 +12,7 @@ defmodule Content.Audio.FollowingTrainTest do
 
       assert Content.Audio.to_params(audio) ==
                {:canned,
-                {"115",
-                 [
-                   "667",
-                   "21000",
-                   "4016",
-                   "21000",
-                   "864",
-                   "21000",
-                   "503",
-                   "21000",
-                   "504",
-                   "21000",
-                   "5005",
-                   "21000",
-                   "505"
-                 ], :audio}}
+                {"115", spaced(["667", "4016", "864", "503", "504", "5005", "505"]), :audio}}
```

Comment:
Suggestion: We could make these lists more concise still by expressing them as

```elixir
~w[667 4016 864 503 504 5005 505]
```

Reply:
Nice idea, will probably handle that in a followup PR